src/scenes/NftDetailPage/NftDetailNote.tsx

```diff
@@ -259,7 +254,16 @@ const StyledTextAreaWithCharCount = styled(TextAreaWithCharCount)<TextAreaProps>
 
   p {
     right: 0;
-    bottom: ${({ footerHeight }) => footerHeight}px;
+    bottom: 0;
+  }
+
+  // We only apply padding to account for footer, which is not fixed on mobile
```

Comment:
Unrelated to this fix, a cleanup bugfix that addressed this:

![image](https://user-images.githubusercontent.com/13339581/155801985-af1fe51b-d200-41b5-951a-599263af9586.png)

![image](https://user-images.githubusercontent.com/13339581/155802025-e0ce0d04-dbdd-4289-b897-3755f0ae79ee.png)

Can open a separate PR if we want but it is very minor lol

Reply:
amazing, great catch!